### PR TITLE
feat: backup LDK activity and blocktank orders

### PR DESCRIPTION
**Pull request by limpbrains**
Because we can't restore LN activity from onchain data or ldk backup we now backup it on every new LN transaction. Same for blocktank orders

### CODE REVIEW

File: src/utils/backup/backups-subscriber.tsx [hunk 1]
```diff
@@ -91,6 +96,59 @@ const EnabledSlashtag = (): ReactElement => {
 		BACKUP_DEBOUNCE,
 	);
 
+	// Attempts to backup ldkActivity anytime remoteMetadataBackupSynced is set to false.
```
Comment (pwltr) [inline src/utils/backup/backups-subscriber.tsx]:
```suggestion
	// Attempts to backup ldkActivity anytime remoteLdkActivityBackupSynced is set to false.
```

File: src/store/actions/backup.ts [hunk 2]
```diff
@@ -396,6 +399,85 @@ export const performMetadataRestore = async ({
...
+	selectedNetwork,
+}: {
+	slashtag: Slashtag;
+	selectedNetwork?: TAvailableNetworks;
+}): Promise<Result<{ backupExists: boolean }>> => {
+	if (!selectedNetwork) {
+		selectedNetwork = getSelectedNetwork();
+	}
+
+	const backupRes = await getBackup<IMetadata>({
```
Comment (pwltr) [inline src/store/actions/backup.ts]: wrong type `IMetadata` ?

File: src/store/actions/backup.ts [hunk 3]
```diff
@@ -396,6 +399,85 @@ export const performMetadataRestore = async ({
...
```
Comment (pwltr) [inline src/store/actions/backup.ts]: wrong type `IMetadata` ?

**Review by pwltr — CHANGES_REQUESTED** [review]

File: src/utils/backup/backups-subscriber.tsx (hunk 1 above)
Comment (limpbrains) [reply src/utils/backup/backups-subscriber.tsx]: fixed

File: src/store/actions/backup.ts (hunk 3 above)
Comment (limpbrains) [reply src/store/actions/backup.ts]: fixed

File: src/store/actions/backup.ts (hunk 2 above)
Comment (limpbrains) [reply src/store/actions/backup.ts]: fixed

**Review by pwltr — APPROVED** [review]

**Pull request merged**